libbindgen/src/codegen/mod.rs

```diff
@@ -324,8 +324,11 @@ impl CodeGenerator for Var {
                 .expr();
             let item = match *val {
                 VarType::Int(val) => {
-                    const_item.build(helpers::ast_ty::int_expr(val))
-                        .build(ty)
+                    let kind = ctx.resolve_item(self.ty()).expect_type().kind();
```

Comment:
#274 may not be that trivial to fix. `clang_EvalResult_getAsInt` returns an `int`, which is signed 32bit integer... If it can give up and hand things to us, we can probably make `get_integer_literal_from_cursor` aware of large integers.